src/UBC Toboggan/Assets/Scripts/playerManager.cs

```diff
@@ -146,22 +146,34 @@ void Update()
         }
     }
 
-    // handles collisions with landing on the ground or getting killed by ground
-    void OnTriggerEnter2D(Collider2D collider) {
-        if (skiTrigger.IsTouching(collider) && collider.tag == "ground") {
+    // handles collisions with landing on the ground or getting killed by ground or falling in a pit
+    void OnTriggerEnter2D(Collider2D collider)
+    {
+        if (skiTrigger.IsTouching(collider) && collider.tag == "ground")
+        {
             grounded = true;
             intAirTime = 0;
             airTime = 0f;
             slideAudio.volume = 0f;
             soundManagerScript.playSound(slideAudio);
         }
-        if (playerTrigger.IsTouching(collider) && collider.tag == "ground") {
+        if (playerTrigger.IsTouching(collider) && collider.tag == "ground")
+        {
+            alive = false;
+            fire.SetFloat("isBoosting", 0f);
+            boostAmount = 0f;
+            StartCoroutine(LoadGameOverScreen());
+        }
+        if (skiTrigger.IsTouching(collider) && collider.tag == "pit")
+        {
```

Comment:
These two if statements result in the same action, so let's combine them into one statement that says 
```cs
if ((playerTrigger.isTouching(collider) && collider.tag == "ground") 
    || (skiTrigger.isTouching(collider) && collider.tag == "pit")) {
     alive = false;
     fire.SetFloat("isBoosting", 0f);
     boostAmount = 0f;
     StartCoroutine(LoadGameOverScreen());
}
```